src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+}
+
+.container {
+  width: 96px;
+  height: auto;
+  margin: 2px;
+  /* border: 1px solid #000; */
+}
+
+.stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-of-type(1) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
Let's reduce the amount of code.
Make one background default and change it to another when necessary.